src/seedu/addressbook/data/person/Tagging.java

```diff
@@ -0,0 +1,52 @@
+package seedu.addressbook.data.person;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.tag.UniqueTagList;
+
+public class Tagging {
+    
+    public final String symbol;
+    public final String name;
+    public final String tagValue;
+    private final StringBuilder result = new StringBuilder();
+    
+    public static final String BREAK = " ";
+    public static final String OPEN_BRACKET = "[";
+    public static final String CLOSE_BRACKET = "]";
+    public static final ArrayList<Tagging> tagHistory = new ArrayList<Tagging>();
```

Comment:
Based on the class diagram given in the exercise, the list of `Tagging` objects should not be in this class. If it was in this class, the diagram would be like this:
![image](https://cloud.githubusercontent.com/assets/1673303/18409597/69111e4e-777d-11e6-92bf-9914c825bd13.png)